membership-attribute-service/app/services/subscription/SubscriptionTransform.scala

```diff
@@ -97,6 +147,22 @@ object SubscriptionTransform {
       case _ => false
     })
 
+  def getSubscriptionsActiveOnOrAfterWithAccounts[P <: AnyPlan: SubPlanReads](
+      onOrAfter: LocalDate,
+  )(catalog: CatalogMap, pids: ProductIds)(
+      subJsons: List[(Account, JsValue)],
+  ): Disjunction[String, NonEmptyList[(Account, Subscription[P])]] =
+    Sequence(
+      subJsons
+        .map { case (account, subJson) =>
+          getSubscription[P](catalog, pids)(subJson).map(s => (account, s))
+        } // why can't I call map or traverse on tuples here to avoid manual (un)packing?
```

Comment:
I haven’t used scalaz a huge amount before – I’m assuming what’s missing here is the right kind of import or magic to get the relevant implicits into scope, but I can’t work it out. Let me know if you know!